January newsletter & december stats (#465)
readthedocs/ethicalads.io: content/posts/newsletter-january-2025.md

tags: newsletter, community, build-in-public
authors: David Fischer
image: /images/posts/2025-january-newsletter.jpg
image_credit: <span>Image created with <a href="https://chatgpt.com/" title="A watercolor-style painting of a winter cityscape with a cyberpunk theme. The setting includes snow-covered streets, glowing neon digital billboards">ChatGPT</a></span>

## New features from the previous month

* We released our new
  [prospectus]({static}../prospectus/ethicalads-advertiser-prospectus.pdf)
  for Q1. It has a price reduction across a few markets and some more information on
  [niche targeting]({filename}../pages/niche-targeting.md).
* The legwork for a new advertiser domain breakdown report was finished up
  and we expect the report to be live in the next few days.
  This report will break down advertiser traffic
  by the domain where the ad was shown. Quite a few advertisers asked for this.
* We got a few more advertisers trying our [niche targeting]({filename}../pages/niche-targeting.md) options.
  This resulted in us building a few additional UX and reporting features for it.

> * We began testing some ads on Reddit in a very small way.
>   This had a couple goals from learning more about what our competitors are doing well
>   so these features can be integrated into our own service
>   as well as trying to get a few new advertisers while doing it.
> 

Reviewer: ```suggestion
* We began testing some ads on Reddit in a very small way.
  This had a couple goals, market research and trying to get a few new advertisers while doing it.   
```

This felt pretty verbose?
